server.js

```diff
@@ -3951,6 +3951,41 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Snap CI build integration.
+// https://snap-ci.com/snap-ci/snap-deploy/branch/master/build_image
+camp.route(/^\/snap(-ci?)\/([^\/]+\/[^\/]+)(?:\/(.+))\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var userRepo = match[2];
+  var branch = match[3];
+  var format = match[4];
+  var url = 'https://snap-ci.com/' + userRepo + '/branch/' + branch + '/build_image.svg';
+
+  var badgeData = getBadgeData('build', data);
+  fetchFromSvg(request, url, function(err, res) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+    try {
+      badgeData.text[1] = res.toLowerCase();
+      if (res === 'Passed') {
+        badgeData.colorscheme = 'brightgreen';
+      } else if (res === 'Failed') {
+        badgeData.colorscheme = 'red';
+      } else {
+        badgeData.text[1] = res.toLowerCase();
```

Comment:
I believe this operation does not change the content of `badgeData.text[1]`, as it was already set to the lowercase version of `res`. Am I wrong?

If I am correct, could you remove the else clause please?